osCam/userconfig/forms.py

```diff
@@ -119,3 +119,21 @@ def __init__(self, *args, **kwargs):
         self.fields['maxSpace'].label = "Max Space"
         self.fields['timeToLive'].label = "Time to Live"
         self.fields['lengthOfRecordings'].label = "Length of Recordings"
+
+class EmailEntryForm( ModelForm ):
+    email = forms.CharField(label='email address', required=False,
+        widget=forms.TextInput(attrs={'size':'80', 'class':"form-control"})
+        )
+    emailKey = forms.CharField(label='email authentication key', required=False,
```

Comment:
can we make it so the email key is hidden? like how passwords are starred out: "*****************"